docs/.vitepress/config.mjs

```diff
@@ -158,6 +158,10 @@ export default defineConfig({
             text: "master",
             link: "https://docs.qgroundcontrol.com/master/en/",
           },
+          {
```

Comment:
FYI, this is right, but if you also want the links in other versions you'll have to also update this in those branches.